service/src/main/java/fish/focus/uvms/plugins/ais/service/AisService.java

```diff
@@ -124,6 +132,14 @@ private void connect() {
         connection.open(host, port, username, password);
     }
 
+    void fetchAssetList () {
+        SearchBranch searchBranch = new SearchBranch(true);
+        List<AssetDTO> assetDTOList =  assetClient.getAssetList(searchBranch);
```

Comment:
getAssetList can throw a RuntimeException that we must deal with since the \@Schedule method should not throw application exceptions.